pkg/signaling/mcast/backend.go

```diff
@@ -0,0 +1,141 @@
+// SPDX-FileCopyrightText: 2025 Adam Rizkalla <ajarizzo@gmail.com>
+// SPDX-License-Identifier: Apache-2.0
+
+package mcast
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"syscall"
+
+	"cunicu.li/cunicu/pkg/crypto"
+	"cunicu.li/cunicu/pkg/log"
+	signalingproto "cunicu.li/cunicu/pkg/proto/signaling"
+	"cunicu.li/cunicu/pkg/signaling"
+	"go.uber.org/zap"
+	"google.golang.org/protobuf/proto"
+)
+
+func init() { //nolint:gochecknoinits
+	signaling.Backends["multicast"] = &signaling.BackendPlugin{
+		New:         NewBackend,
+		Description: "Multicast",
+	}
+}
+
+type Backend struct {
+	signaling.SubscriptionsRegistry
+
+	conn       *net.UDPConn
+	mcast_addr *net.UDPAddr
```

Comment:
Please use camelCase for variable names:

https://go.dev/wiki/CodeReviewComments#variable-names

```suggestion
	mcastAddr *net.UDPAddr
```